.storybook/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
do we need this? we should remove

Reply:
this was a suggestion for running storybook for an issue with storybook running in type=module project

Reply:
yea, we may need this. I tried debugging a bit but it seems required.